.github/bin/format-package-json.mjs

```diff
@@ -51,6 +51,9 @@ const formatted = {};
 
 	formatted.license = packageJSONInfo.license;
 	delete packageJSONInfo.license;
+	if (formatted.license !== 'CC0-1.0' && formatted.license !== 'MIT') {
+		formatted.license = 'MIT';
+	}
```

Comment:
Is this intended?

Reply:
Not really, just copy/paste from the last string on line 54.
I will change to `CC0-1.0` which is the most common license in plugins here.